src/main/rtspServer.ts

```diff
@@ -0,0 +1,45 @@
+import { log } from '@/main/logger';
+import { app, ipcMain } from 'electron';
+import { ExecaChildProcess, execa } from 'execa';
+import path from 'path';
+import { isDev } from '@/main/isDev';
+
+interface RtspProcess {
+  process: ExecaChildProcess;
+  wsPort: number;
+}
+
+const rtspServers: Map<number, RtspProcess> = new Map();
+
+// Stack array of ports from (9000 to 9060) to use for rtsp servers
+const ports = Array.from({ length: 61 }, (_, i) => i + 9000);
+
+ipcMain.handle('rtsp_start', (event, url: string) => {
```

Comment:
What is the difference between rtsp_start here and the value assigned to the constants in preload?

Reply:
None, I just forgot to use the exported constants, I fixed it, good catch!